app/models/prompt.rb

```diff
@@ -117,13 +119,13 @@ def correct_number_of_tags
               ts("none") :
               "(#{tag_count}) -- " + taglist.collect(&:name).join(ArchiveConfig.DELIMITER_FOR_OUTPUT)
           if allowed == 0
-            errors.add(:base, ts("^#{prompt_type}: Your #{prompt_type} cannot include any #{tag_type} tags, but you have included %{taglist}.",
+            errors.add(:base, ts("^#{prompt_type}: Your #{prompt_type} cannot include any #{tag_label} tags, but you have included %{taglist}.",
```

Comment:
According to a comment by ticking (https://github.com/otwcode/otwarchive/pull/3526#issuecomment-475897070), lines 108/109 and 150/151 also need `tag_type` changed to `tag_label`.

https://github.com/otwcode/otwarchive/blob/ab0697a2b42c839e5dcb484d03ce50caa0ba1932/app/models/prompt.rb#L108-L109

https://github.com/otwcode/otwarchive/blob/ab0697a2b42c839e5dcb484d03ce50caa0ba1932/app/models/prompt.rb#L150-L151

I think we also need the change on 182/184:

https://github.com/otwcode/otwarchive/blob/ab0697a2b42c839e5dcb484d03ce50caa0ba1932/app/models/prompt.rb#L182-L184